examples/tools.html

```diff
@@ -0,0 +1,12 @@
+<script type="module">
+    import {HanskenClient} from './src/hansken-js.js';
+    const hansken = new HanskenClient('/gatekeeper', '/keystore');
+
+    hansken.scheduler().tools()
```

Comment:
I wouldn't expect the retrieval of tools at the scheduler. I would expect something like `hansken.tools()`

Reply:
Discussed, ok